src/ui/common/src/components/pages/workflows/index.tsx

```diff
@@ -308,10 +305,11 @@ const WorkflowsPage: React.FC<Props> = ({ user, Layout = DefaultLayout }) => {
           onChangeRowsPerPage={onChangeRowsPerPage}
           savedRowsPerPage={getRowsPerPage()}
           sortColumns={sortColumns}
-          defaultSortConfig={{
-            sortColumn: sortColumns[0],
-            sortType: SortType.Descending,
-          }}
+          // Disabled until ENG-3096
```

Comment:
Nit: Add a `TODO:` in this comment. This helps us finding and clean up `TODO`s if these follow up didn't get in due to other priorities.